src/main/flight/mixer.c

```diff
@@ -561,6 +578,76 @@ static void airplaneMixer(void)
 }
 #endif
 
+/*
+ * float in range [-PI/2:+PI/2]
+ */
+float getTiltAngle(void) {
+    float angleTilt = 0;
+    //get estimated position of the tilting servo
+    if (rcData[AUX1] > rxConfig->midrc) {
+        angleTilt = rcData[AUX1];
+    } else {
+        angleTilt = rcData[PITCH];
+    }
+    //find max rx range
+    float halfMaxRange = (rxConfig->maxcheck - rxConfig->mincheck) / 2.0;
+    //find how much the Servo WILL be tilted from -1 to 1
+    float angle = (angleTilt - rxConfig->midrc) / halfMaxRange;
+    //convert to radiant
+    return angle * M_PIf / 2;
+}
+
+void servoTilting(void) {
+    float actualTilt = getTiltAngle();
+
+    //do we need to invert the Servo direction?
+    if (servoConf[TILTING_SERVO].rate & 1){
+        actualTilt *= -1;
+    }
+
+    //limit servo escursion
+    actualTilt = constrainf( actualTilt, degreesToRadians(servoConf[TILTING_SERVO].minLimit), degreesToRadians(servoConf[TILTING_SERVO].maxLimit) );
+
+    //remap input value (RX limit) to output value (Servo limit), also take into account eventual non-linearity of the full range
+    if (actualTilt > 0){
+        actualTilt = mapf(actualTilt, 0, +M_PI/2, servoConf[TILTING_SERVO].middle, servoConf[TILTING_SERVO].max);
+    }else{
+        actualTilt = mapf(actualTilt, -M_PI/2, 0, servoConf[TILTING_SERVO].min, servoConf[TILTING_SERVO].middle);
+    }
+
+    //just to be sure
+    int16_t outputPwm = constrain( actualTilt, servoConf[TILTING_SERVO].min, servoConf[TILTING_SERVO].max );
+
+    //and now write it!
+    pwmWriteServo(TILTING_SERVO, outputPwm);
+}
+
+void mixTilting(void) {
+    float angleTilt = getTiltAngle();
+
+    //do heavy math only one time
+    float tmpCosine = cosf(angleTilt);
+    float tmpSine = sinf(angleTilt);
+
+    if (currentMixerMode == MIXER_QUADX_TILT_THRUST || currentMixerMode == MIXER_QUADX_TILT_ALL) {
+        // compensate the throttle because motor orientation
+        rcCommand[THROTTLE] += (rcCommand[THROTTLE]-1000) * tmpSine;
+    }
+
+    if (currentMixerMode == MIXER_QUADX_TILT_COS || currentMixerMode == MIXER_QUADX_TILT_ALL) {
+
+        //***** quick and dirty compensation to test *****
```

Comment:
yes, it's the mathematical throttle compensation if you do vector math, but was wrong as it wasn't taking into account that to hover your base value was 1000 (on a range of 1000-2000), while realistically it changes quad to quad and is more around 1500

newest code use by default 50% (so 1500) and it can be changed by configurator. Also, even if you enable the tilt configuration, now you have to manually enable its feature